docs/release.md

```diff
@@ -105,12 +106,12 @@ The last step of the release process creates artifacts for the new version of th
 
 ## Troubleshooting
 
-1. `unknown revision internal/coreinternal/v0.45.0` -- This is typically an indication that there's a dependency on a new module. You can fix it by adding a new `replaces` entry to the `go.mod` for the affected module. 
+1. `unknown revision internal/coreinternal/v0.55.0` -- This is typically an indication that there's a dependency on a new module. You can fix it by adding a new `replaces` entry to the `go.mod` for the affected module. 
 
 ## Release schedule
 
 | Date       | Version | Release manager |
-|------------|---------|-----------------|
+| ---------- | ------- | --------------- |
```

Comment:
Did you format this automatically using some tool? The tool I use would replace this space with a dash, but if yours format it this way, I'm fine with it.

Reply:
@jpkrohling yes, its the markdown formatter im using in vscode